components/StarSearchWidgets/LotteryFactorWidget.tsx

```diff
@@ -0,0 +1,26 @@
+import LotteryFactorChart from "components/Repositories/LotteryFactorChart";
+import { useRepositoryLottoFactor } from "lib/hooks/api/useRepositoryLottoFactor";
```

Comment:
This is a thin wrapper component on the lottery factor chart that includes the same hook on the repo page.

This is probably the pattern we want to adopt for other widgets.